packages/block-editor/src/components/rich-text/index.js

```diff
@@ -400,6 +400,8 @@ class RichTextWrapper extends Component {
 				__unstableUndo={ undo }
 				style={ style }
 				preserveWhiteSpace={ preserveWhiteSpace }
+				start={ start }
+				reversed={ reversed }
```

Comment:
Both these props are already added further down to `Editable`.

Reply:
Would it be possible to make that work for native? Ideally all not rich text related props are directly passed to the editable element.

Reply:
So we don't have editable on native, and all of the props need to go trough rich text.

One thing that I was thinking was to create an TagAttributes property that will go along with TagName prop and controlled by the blocks.

What do you think @ellatrix ?

Reply:
Would it be possible to create an `Editable` on native?

Reply:
It will be possible, but it will take some considerable work to get there, and out of scope for this PR.

We basically will need to make our Native Aztec rich text component to be the Editable and respect all the API for it.

@koke and @hypest may have more thoughts about this, and when to give it to priority.

Reply:
I talked with @ellatrix and we agree to keep this as it is, and tackle it on a separate PR.